integration/noderunner/noderunner_test.go

```diff
@@ -86,20 +75,15 @@ func TestCanStartStandaloneObscuroHostAndEnclave(t *testing.T) {
 		panic(err)
 	}
 
-	go func() {
-		err := enclavecontainer.NewEnclaveContainerFromConfig(enclaveConfig).Start()
-		if err != nil {
-			panic(err)
-		}
-	}()
+	inMemNode, _ := node.NewInMemNode(nodeCfg)
```

Comment:
We should check the error here, else risk nil pointering below and making it harder to debug in future.

Reply:
I think if the method returns an error type the caller should assume it might return an error. Relying on devs checking the usages when they update it seems like a pointless risk.

If you don't think it needs it then scrap the error return maybe?

Reply:
Yeah makes sense 👍